src/validations/rules/ssp.sch

```diff
@@ -2452,6 +2452,22 @@
                 role="warning"
                 test="//oscal:responsible-party[oscal:party-uuid = current()/@uuid]">Each person should have a responsibility.</sch:assert>
         </sch:rule>
+        <sch:rule 
+            context="oscal:location[oscal:prop[@value eq 'data-center']]"
+            doc:guide-reference="Guide to OSCAL-based FedRAMP System Security Plans §4.23">
+            <sch:assert 
+                diagnostics="data-center-country-code-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP System Security Plans §4.23"
+                id="data-center-country-code"
+                role="warning"
+                test="oscal:address/oscal:country">Each data center address must contain a country.</sch:assert>    
```

Comment:
Suggest "Each data center address must contain a country-code."

Reply:
The OSCAL XML Schema constrains this (and an empty `<country>` will throw a somewhat unhelpful  Value '' is not facet-valid with respect to pattern '\S(.*\S)?' for type '#AnonType_countryoscal-metadata-address-ASSEMBLY'). The XML Schema does not constrain it to ISO-3166 country codes.

Reply:
I got my information on 3166-2 from here:
https://pages.nist.gov/OSCAL/reference/1.0.1/complete/xml-reference/#/catalog/metadata/location/address/country

The way I read the wiki page on ISO 3166-2 is that all combinations of AA through ZZ are available but only some are currently assigned.  So the error message is true but might be unhelpful if someone typoed with 'UL' for instance.  That would be valid to oscal but not a currently assigned ISO 3166-2 value.
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2

Anyway, my main point to this comment is that the rule and diagnostic are both directed at the country element not the code.  So I will change the ids and assertations to match that thought.